### PR TITLE
Please review my tasks 

### PR DESCRIPTION
**Pull request by Bob-TheTester**
Hello, Mr. Dung, 

I am done with my task. Can you take a look ? 

Bob

### CODE REVIEW

File: circle.h [hunk 1]
```diff
@@ -10,6 +10,9 @@ class Circle {
 	Circle & print() {
 		cout << "R:" << r << endl;
 	}
+	void bob_function(int blah) {
+		cout << "Bob say, hello" << endl;
```
Comment (dungdna2000) [inline circle.h:14]: WTF bob! You cannot cout like this!

**Review by dungdna2000 — CHANGES_REQUESTED** [review]
Bob, pls take a look at my comments , fix them , commit and try again

File: circle.h (hunk 1 above)
Comment (Bob-TheTester) [reply circle.h:14]: Sorry sir, my bad. I'll fix this immediately. Please don't be angry with me!

**Review by dungdna2000 — APPROVED** [review]
Bob, good job but remember next time, don't make this stupid code mistake again, ever!!!

**Pull request merged**